source/PlayState.hx

```diff
@@ -4083,7 +4087,7 @@ class PlayState extends MusicBeatState
 
 		if (generatedMusic)
 		{
-			notes.sort(FlxSort.byY, ClientPrefs.downScroll ? FlxSort.ASCENDING : FlxSort.DESCENDING);
+			notes.sort(FlxSort.byY, (ClientPrefs.downScroll) ? FlxSort.ASCENDING : FlxSort.DESCENDING);
```

Comment:
unnecesary change ngl

Reply:
Indeed, I noticed the note trails were still inconsistent in this engine and tried to do something and that was a left over of the experiments. In practice, it doesn't really do anything.